src/components/autocomplete/js/autocompleteController.js

```diff
@@ -207,6 +207,17 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
     elements.$    = getAngularElements(elements);
   }
 
+	/**
+	 * Finds the correct element that will determine the width
+	 * @returns {*}
+	 */
+	function getWrapTarget() {
+    for (var element = $element; element.length; element = element.parent()) {
+      if (angular.isDefined(element.attr('md-autocomplete-wrap-override'))) return element[ 0 ];
+    }
+    return $element.find('md-autocomplete-wrap')[0];
```

Comment:
You should fix the indentation here and above. Using 2 spaces for one indentation.